ReSharper.FSharp/src/FSharp/FSharp.Common/src/Settings/FSharpOptions.fs

```diff
@@ -191,7 +191,12 @@ type FSharpTypeHintOptions =
       [<SettingsEntry(PushToHintMode.PushToShowHints,
                       DescriptionResourceType = typeof<Strings>,
                       DescriptionResourceName = nameof(Strings.FSharpTypeHints_LocalBindings_Description))>]
-      mutable ShowTypeHintsForLocalBindings: PushToHintMode }
+      mutable ShowTypeHintsForLocalBindings: PushToHintMode
+
+      [<SettingsEntry(PushToHintMode.PushToShowHints,
+                      DescriptionResourceType = typeof<Strings>,
+                      DescriptionResourceName = nameof(Strings.FSharpTypeHints_OtherPatterns_Description))>]
+      mutable ShowTypeHintsForOtherPatterns: PushToHintMode }
```

Comment:
Nit-picking: `ShowTypeHintsForOtherPatterns` sounds a bit tautological in the context of `FSharpTypeHintOptions`. Could it maybe be something like `ShowOtherPatterns`/`OtherPatternsMode`? Ideally, this should be inline with the other settings, so I'm not sure if it's feasible to change it without breaking existing settings or unneeded migrations.